src/containers/Chat/ChatMessages/ChatInput/ChatInput.tsx

```diff
@@ -52,46 +56,76 @@ export const ChatInput: React.SFC<ChatInputProps> = ({ onSendMessage }) => {
     );
   }
 
+  const handleChange = (editorState: any) => {
+    // Get the markdown equivalent for this text.
+    // BUG: When highlighting over text and bolding, the cursor must move at least once in order for the changes to take effect.
+    let markdownString = draftToMarkdown(convertToRaw(editorState.getCurrentContent()));
```

Comment:
The `getCurrentContent` method causes two tests to fail. I tried looking up the error but I haven't had much success with it. Will continue trying to debug later, but would love to get help on passing this.

Current error: `TypeError: editorState.getCurrentContent is not a function`